mongo_connector/command_helper.py

```diff
@@ -75,3 +75,8 @@ def map_collection(self, db, coll):
             return tuple(ns.split('.', 1))
         else:
             return None, None
+
+    def revert_namespace(self, namespace):
+        full_type = self.dest_mapping.keys()[self.dest_mapping.values().index(namespace)].split(".", 1)
+        return full_type[0], full_type[1]
```

Comment:
It's probably better to create another map like `dest_mapping` called `dest_mapping_inverse` (or something like that) in `__init__` and use that here, since we're potentially calling this method for every document changed in the oplog.